docs/sitegen.yaml

```diff
@@ -52,26 +52,20 @@ backend:
         type: "image"
         value: "images/helidon_logo_dark.svg"
       items:
+        - type: "PAGE"
+          title: "Introduction"
+          source: "about/introduction.adoc"
+          glyph:
+            type: "icon"
+            value: "widgets"
+        - type: "PAGE"
+          title: "Helidon CLI"
+          source: "about/cli.adoc"
+          glyph:
+            type: "icon"
+            value: "widgets"
         - type: "GROUPS"
           items:
-          - type: "GROUP"
-            title: "Introduction"
-            dir: "about"
-            items:
-              - type: "MENU"
-                title: "About"
-                glyph:
-                  type: "icon"
-                  value: "assistant"
-                sources:
-                  - "overview.adoc"
-                  - "introduction.adoc"
-                  - "prerequisites.adoc"
```

Comment:
Some version of `prerequisistes.adoc` should be preserved in a top level `getstarted.adoc`

Reply:
We are trying to compact the left panel. Prerequisities are not a part of introduction page and can be referenced as `http://xxx/about/introduction#_prerequisites)`